### PR TITLE
Improve code structure in relation to else / if statements.

### PR DESCRIPTION
**Pull request by nuzil**
### Description
This PR improves code structure for usage of if / else statements

### Fixed Issues (if relevant)
No, code improvements

### Manual testing scenarios
No, code improvements

### Contribution checklist
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

File: app/code/Magento/AdminNotification/Block/Window.php [hunk 1]
```diff
@@ -98,10 +98,9 @@ protected function _getLatestItem()
     {
         if ($this->_latestItem == null) {
             $items = array_values($this->_criticalCollection->getItems());
+            $this->_latestItem = false;
             if (count($items)) {
                 $this->_latestItem = $items[0];
```
Comment (orlangur) [inline app/code/Magento/AdminNotification/Block/Window.php:103]: Ternary operator would fit better here.

File: app/code/Magento/AdvancedPricingImportExport/Model/Export/AdvancedPricing.php [hunk 2]
```diff
@@ -626,9 +626,8 @@ protected function _getCustomerGroupById(
     ): string {
         if ($allGroups !== 0) {
             return ImportAdvancedPricing::VALUE_ALL_GROUPS;
-        } else {
-            return $this->_groupRepository->getById($groupId)->getCode();
         }
+        return $this->_groupRepository->getById($groupId)->getCode();
```
Comment (orlangur) [inline app/code/Magento/AdvancedPricingImportExport/Model/Export/AdvancedPricing.php:630]: Ternary operator would fit better here.

File: app/code/Magento/Analytics/Model/Config/Backend/Enabled.php [hunk 3]
```diff
@@ -67,12 +67,7 @@ public function afterSave()
         try {
             if ($this->isValueChanged()) {
                 $enabled = $this->getData('value');
-
-                if ($enabled) {
-                    $this->subscriptionHandler->processEnabled();
-                } else {
-                    $this->subscriptionHandler->processDisabled();
-                }
+                $enabled ? $this->subscriptionHandler->processEnabled() : $this->subscriptionHandler->processDisabled();
```
Comment (orlangur) [inline app/code/Magento/Analytics/Model/Config/Backend/Enabled.php:70]: Ternary operator is NOT really suitable here as it decreases code readability. it is better to be used in assignments only.

**Review by orlangur — CHANGES_REQUESTED** [review]

**Comment by magento-engcom-team** [comment]
Hi @nuzil. Thank you for your contribution. 
Changes from your Pull Request will be available with the upcoming 2.3.0 release.

**Pull request merged**